www/js/control/ProfileSettings.jsx

```diff
@@ -21,24 +25,25 @@ const ProfileSettings = () => {
     const { colors } = useTheme();
 
     // get the scope of the general-settings.js file
-    const mainControlEl = document.getElementById('main-control').querySelector('ion-view');
-    const settingsScope = angular.element(mainControlEl).scope();
+    const mainControlEl = document.getElementById('main-control');
+    const settingsScope = angular.element(mainControlEl.querySelector('profile-settings')).scope();
+    console.log("settings scope", settingsScope);
+    
     // grab any variables or functions we need from it like this:
-    const { settings, logOut, viewPrivacyPolicy,
-        fixAppStatus, forceSync, openDatePicker,
-        eraseUserData, refreshScreen, endForceSync, checkConsent, 
-        dummyNotification, invalidateCache, showLog, showSensed,
-        parseState, userDataSaved, userData, ui_config } = settingsScope;
+    const { showLog, showSensed } = settingsScope;
```

Comment:
significantly less dependent on the `settingsScope` variables -- which means decreasing dependence on angular code, and approaching being able to use React routing